notebooks/data-sources/oc-github-repo/time_to_merge_model.ipynb

```diff
@@ -0,0 +1,1123 @@
+{
```

Comment:
<p>Did you consider looking at confusion matrix as well? ie, are these errors being made on the boarders of these ordinal classes or are the all over the place? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/331/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
@MichaelClifford that's a good idea. We will also try inspecting the confusion matrices for each of the models. 👍

Reply:
<p>On plotting the confusion matrices, it seems like the Naive Bayes and SVM models are biased towards a few classes. The random forest and XG Boost models seem to mainly have mis-classifications close to the borders classes of the ordinal classes. </p><p> </p><p> </p>